Code/Editor/QtViewPaneManager.h

```diff
@@ -245,9 +245,9 @@ class EDITOR_CORE_API QtViewPaneManager
 
     QPointer<AzQtComponents::FancyDocking> m_advancedDockManager;
 
-    using EditorComponentModeNotificationBusImpl = AzToolsFramework::ComponentModeFramework::EditorComponentModeNotificationBusImpl;
-    EditorComponentModeNotificationBusImpl m_componentModeNotifications; //!< Helper for EditorComponentModeNotificationBus so
-                                                                         //!< QtViewPaneManager does not need to inherit directly from it. */
+    AZStd::unique_ptr<ViewportEditorModeNotificationsBusImpl>
+        m_componentModeNotifications; //!< Helper for EditorComponentModeNotificationBus so
+                                                                                            //!< QtViewPaneManager does not need to inherit directly from it. */
```

Comment:
fixed the indentation 👍 

re: `unique_ptr`, the impl is in the source file so forward declared (it's the class for wrapping the bus instead of inheriting it to avoid those link errors).